### ISSUE TITLE
no-return-wrap doesn't work for arrow functions without braces

### ISSUE THREAD

**Issue by cherryblossom000** [xjamundx/eslint-plugin-promise#193]
### Description

no-return-wrap doesn't work for arrow functions without braces.

### Steps to Reproduce

1.

`.eslintrc.json`
```json
{
  "env": {"es6": true},
  "plugins": ["promise"],
  "rules": {
    "promise/no-return-wrap": 2
  }
}
```

`test.js`
```js
promise.then(() => Promise.resolve())
promise.then(() => Promise.reject())
```

2. Run `eslint test.js`.

**Expected behavior:** The errors `Avoid wrapping return values in Promise.resolve` and `Expected throw instead of Promise.reject`.

**Actual behavior:** No errors.

### Versions

- Node version: 12.18.2
- ESLint version: 7.3.1
- eslint-plugin-promise version: 4.2.1
